crypto/s2n_ecc.c

```diff
@@ -26,7 +26,7 @@
 #include "utils/s2n_safety.h"
 #include "utils/s2n_mem.h"
 
-#define TLS_EC_CURVE_TYPE_NAMED 3
+#define TLS_EC_CURVE_TYPE_NAMED 2
```

Comment:
is this change related to this PR?

Reply:
This is related to the other elliptical curves, we currently support only 2 NIST curves (`secp256r1`, `secp384r1`  and not 3.